tests/Auth0.ManagementApi.IntegrationTests/LogStreamsTests.cs

```diff
@@ -0,0 +1,135 @@
+﻿using Auth0.Core.Exceptions;
+using Auth0.ManagementApi.Models;
+using Auth0.Tests.Shared;
+using FluentAssertions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Auth0.ManagementApi.IntegrationTests
+{
+    public class LogStreamsTests : TestBase, IAsyncLifetime
+    {
+        private ManagementApiClient _apiClient;
+        private List<LogStream> createdStreams = new List<LogStream>();
+
+        public async Task InitializeAsync()
+        {
+            string token = await GenerateBruckeManagementApiToken();
+            _apiClient = new ManagementApiClient(token, GetVariable("BRUCKE_MANAGEMENT_API_URL"));
+        }
+
+        public Task DisposeAsync()
+        {
+            // Clean up any log stream entities on the tenant after every test executes
+            var deleteTasks = createdStreams.Select(stream => _apiClient.LogStreams.DeleteAsync(stream.Id));
+
+            return Task.WhenAll(deleteTasks.ToArray()).ContinueWith(_ =>
+            {
+                _apiClient.Dispose();
+            });
+        }
```

Comment:
`DisposeAsync` is called after every test, including when an exception occurred or assertion failed. So here we get an opportunity to clean up any log streams that were created as part of the test run, even when the test fails.